<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveView</class>
 <widget class="QWidget" name="ReceiveView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>接收文件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Settings">
     <property name="title">
      <string>接收设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ListenIP">
        <property name="text">
         <string>监听IP地址：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_ListenIP">
        <property name="text">
         <string>0.0.0.0</string>
        </property>
        <property name="placeholderText">
         <string>0.0.0.0表示监听所有网络接口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_ListenPort">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_ListenPort">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_SavePath">
        <property name="text">
         <string>保存位置：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_SavePath">
        <property name="placeholderText">
         <string>选择文件保存的目录</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_Browse">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ReceivedFiles">
     <property name="title">
      <string>接收文件列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget_ReceivedFiles">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>文件名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>大小</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>发送者</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>状态</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>完成时间</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Status">
     <property name="title">
      <string>当前状态</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ReceiveStatus">
        <property name="text">
         <string>未开始监听</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_OpenFolder">
       <property name="text">
        <string>打开文件夹</string>
       </property>
       <property name="icon">
        <iconset theme="folder-open"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ClearList">
       <property name="text">
        <string>清空列表</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_StartStop">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>开始监听</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
